### DISCUSSION TITLE
uf2 file for a basic synth?

### DISCUSSION THREAD

**Discussion by eurorackworld** [DatanoiseTV/PicoDSP-Hardware#5] [Q&A] [answered]
Hi , I've read all the great stuff about the PicoADK and now have 2 boards at home available. So the first important step has been taken. 

Now it's time to get a sample working :), but I'm not the best dev/github expert. I'm currently doing my first firmware creation for another project and trying to do the same with all the examples for this project.  But meanwhile I'm wondering if there is a basic uf2 file available for download to just have a test of a working synth? Just firmware like for 4 pots controlling some parameters of a synth would already be great.

As I probably need some time before I get either this working or via using Vult (not even sure if using that is mandatory, I still need to read a lot about it;)), so if someone has compiled file somewhere I can use that would be great!

**Comment by DatanoiseTV** [comment, marked as answer]
Hey, thanks for you message. Indeed, there is a synth demo using 4 potentiometers in the firmware repository. You can find the UF2 file in the "releases" page of picoadk-firmware-template.

**Reply by eurorackworld** [reply]
ah great, found it! Checked so many pages, but not that release page. Thanks!
